src/components/header/Header.js

```diff
@@ -2,12 +2,15 @@ import React from "react";
 import "./Header.css";
 import {Fade} from "react-reveal";
 import {greeting, workExperiences} from "../../portfolio";
+import { headroom } from "headroom";
 
 function Header() {
   const exp = workExperiences.viewExperiences;
   return (
-    <Fade top duration={1000} distance="20px">
-    <div>
+  <Fade top duration={1000} distance="20px">
+    <script src="https://unpkg.com/headroom.js"></script>
```

Comment:
You should not use script tag here.

Reply:
Instead here
```html
<Headroom>
```
Should be coming.

Reply:
but the import headroom from headroom is not working

Reply:
I tried to guide you in my comment on #174 too, take a look at Headroom's documentation.

Reply:
i added and followed all the steps and this error occurred
![Capture](https://user-images.githubusercontent.com/67779397/87015492-1f9bc900-c1eb-11ea-81c4-294ae77e8cb2.PNG)

Reply:
Change Headroom.js to `"Headroom"` @archit0210

Reply:
i did, there is no change in error

Reply:
okay, now is this correct?
![Uploading 1.PNG…]()

Reply:
okay, now is this correct?
![1](https://user-images.githubusercontent.com/67779397/87022363-021f2d00-c1f4-11ea-8a6a-c33c6c581aec.PNG)

Reply:
Umm, why are you initializing Headroom, I don't remember exactly but it isn't as complicated as your implementation, you just need to import Headroom and wrap the Header component in it. 
Headroom will make the header sticky itself!

Reply:
![2](https://user-images.githubusercontent.com/67779397/87030712-a2c71a00-c1ff-11ea-82b7-c06cf0573423.PNG)
oh sorry i forgot, i have wrapped header component...

Reply:
i tried restarting and checked the names of files but couldn't resolve!!

Reply:
No